src/applications/vaos/tests/mocks/setup.js

```diff
@@ -244,7 +244,8 @@ export async function setPreferredDate(store, preferredDate) {
 }
 
 export async function setExpressCareFacility({ store }) {
-  store.dispatch(fetchExpressCareWindows());
+  const windowsThunk = fetchExpressCareWindows();
```

Comment:
I think that the previous call doesn't fully wait for all the windows actions to dispatch, because the code that normally waits for that is in component for the express care app section, rather than individual pages.